tests/Type/DefinitionTest.php

```diff
@@ -515,18 +539,6 @@ public function testIdentifiesOutputTypes() : void
         }
     }
 
-    /**
-     * @see it('prohibits nesting NonNull inside NonNull')
-     */
-    public function testProhibitsNestingNonNullInsideNonNull() : void
-    {
-        $this->expectException(InvariantViolation::class);
-        $this->expectExceptionMessage(
-            'Expected Int! to be a GraphQL nullable type.'
-        );
-        Type::nonNull(Type::nonNull(Type::int()));
-    }
-
```

Comment:
This might still be useful to check (even if we have core exception vs a custom one). Because one day we can change something in the `NonNull` type signature and there won't be any test to fail.

Reply:
Do you mind if we revisit that some other time? I'm sure I can modify this test to catch the core exception, but I would rather not dive down that rabbit hole right now if there is not any pressing need for it.